discretisedfield/tests/test_field.py

```diff
@@ -1935,14 +1945,36 @@ def value_fun(point):
         assert np.allclose(f.integrate(d, cumulative=True).diff(d).array, f.array)
         assert np.allclose(f.diff(d).integrate(d, cumulative=True).array, f.array)
 
-    # Exceptions
-    with pytest.raises(ValueError):
+    # 1d mesh
+    mesh = df.Mesh(p1=0, p2=10e-9, n=5)
+    field = df.Field(mesh, nvdim=1, value=lambda p: p)
+    assert np.allclose(
+        field.integrate("x", cumulative=True).array,
+        [1e-18, 5e-18, 13e-18, 24e-18, 41e-18],
+    )
+
+    # 2d mesh with one cell in integration direction
+    mesh = df.Mesh(p1=(0, 0), p2=(10, 1), cell=(1, 1))
+    f = df.Field(mesh, nvdim=2, value=(2, 2.5))
+    f_int = f.integrate("y", cumulative=True)
+    assert f_int.mesh == f.mesh
+    assert np.allclose(f_int.mean(), (1, 1.25))
+
+
+def test_integrate_exceptions():
```

Comment:
Looks good, shall we also check for a list of directions as this may be a common error? I realise you are already testing for an `int` type.

Reply:
Good idea. Could you please suggest a test for this.